understanding/non-text-content.html

```diff
@@ -183,82 +170,57 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
                   
-                  <dt>Goal</dt>
-                  <dd>Non-text information is available to more people.</dd>
+                  <dt>目標</dt>
+                  <dd>非テキスト情報をより多くの人々が利用できるようにする。</dd>
```

Comment:
気になった点が2つあって、

1. 「非テキスト情報」は「非テキストの情報」でなくてよいか
non-text content は「非テキストコンテンツ」なのですが、non-text contrastは「非テキストのコントラスト」としていて「の」が入ります。後者は contrast of non-text content のニュアンスだろうということで「の」が入っています。ここでも information of non-text content の意味であって、「の」が入るほうになるような気がしています。

2. 「…する」という訳し方で良いのか
「できるようにする」となっていますが、原文には make や let は使われていません。
他の達成基準の解説では、この Goal 部分は make ... や Let という文になっていることが多いのですが、ここはそうではありません。原文のこの書き分けが意図したものかどうかは謎なのですが、ともあれ原文は使役のニュアンスではないので、そこは素直に訳した方が良いように思います。

素直にやるとこうでしょうか。

```suggestion
                  <dd>非テキストの情報を、より多くの人々が利用できる。</dd>
```

もしくはこう……?

```suggestion
                  <dd>非テキストの情報が、より多くの人々に利用できるようになっている。</dd>
```